app/src/main/java/io/lbry/browser/ui/wallet/WalletFragment.java

```diff
@@ -397,19 +397,6 @@ public void onClick(View view) {
             }
         });
 
-        inputSendAddress.setOnFocusChangeListener(new View.OnFocusChangeListener() {
-            @Override
-            public void onFocusChange(View view, boolean hasFocus) {
-                inputSendAddress.setHint(hasFocus ? getString(R.string.recipient_address_placeholder) : "");
-            }
-        });
-        inputSendAmount.setOnFocusChangeListener(new View.OnFocusChangeListener() {
-            @Override
-            public void onFocusChange(View view, boolean hasFocus) {
-                inputSendAmount.setHint(hasFocus ? getString(R.string.zero) : "");
-                inlineBalanceContainer.setVisibility(hasFocus ? View.VISIBLE : View.INVISIBLE);
-            }
-        });
```

Comment:
Why were these lines removed? They perform bits of functionality when the input views gain focus.

Reply:
Because it's bassiclly what it's causing the keyboard doesn't show properly, another way to fix it it's by adding a hint for the wallet card resource.

Reply:
This code is required functionality to change the hint when the input view gains focus, so a better fix would be to explicitly show the soft keyboard when `hasFocus` is true for the corresponding view.

Reply:
uhm... i'll try again then!

Reply:
it's done! i did it without deleting those lines.